.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - node
+script: npm run test:coverage
```

Comment:
Hey Esther, can you take out this line of code. We have not setup test coverage yet thanks